device-connectors/src/testflinger_device_connectors/devices/zapper_kvm/README.md

```diff
@@ -0,0 +1,49 @@
+# Zapper KVM
+
+Zapper-driven provisioning method that makes use of KVM assertions and actions.
+
+## Ubuntu Desktop / Server
+
+Support for vanilla Ubuntu is provided by [autoinstall](https://canonical-subiquity.readthedocs-hosted.com/en/latest/intro-to-autoinstall.html). Supported Ubuntu versions are:
+
+- Desktop >= 23.04
+- Server >= 20.04
+
+Unless specified via _autoinstall_ storage filter, the tool will select the largest storage device on the DUT. See [supported layouts](https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#supported-layouts) for more information.
+
+### Job parameters
+
+- __url__: URL to the image to install
+- __username__: username to configure
+- __password__: password to configure
+- **storage_layout**: can be either `lvm`, `direct`, `zfs` or `hybrid` (Desktop 23.10+)
+- **storage_password** (optional): only available with *storage_layout=lvm*, it's the password for LUKS encryption
```

Comment:
This one is slightly worrisome from an automation standpoint. Do we currently, or eventually have plans to add something that can automatically enter this when the system is rebooted?  I could see how zapper might have a chance at doing that, but if it's not going to, then maybe it's not an option we should provide because it could make the system unusable until someone reinstalls it.

Reply:
Good point, we're not there yet.